tests/unit/backend/layers/common/base_test.py

```diff
@@ -0,0 +1,200 @@
+import copy
+import os
+import typing
+import unittest
+from dataclasses import dataclass
+from typing import List, Optional
+from unittest.mock import Mock
+
+from backend.layers.business.business import BusinessLogic
+from backend.layers.common.entities import (
+    CollectionMetadata,
+    CollectionVersion,
+    DatasetMetadata,
+    DatasetStatusGeneric,
+    DatasetStatusKey,
+    Link,
+    OntologyTermId,
+)
+from backend.layers.persistence.persistence import DatabaseProvider
+from backend.layers.persistence.persistence_mock import DatabaseProviderMock
+from backend.layers.thirdparty.crossref_provider import CrossrefProviderInterface
+from backend.layers.thirdparty.s3_provider_interface import S3ProviderInterface
+from backend.layers.thirdparty.step_function_provider import StepFunctionProviderInterface
+from backend.layers.thirdparty.uri_provider import FileInfo, UriProviderInterface
+
+
+@dataclass
+class DatasetStatusUpdate:
+    status_key: DatasetStatusKey
+    status: DatasetStatusGeneric
+
+
+@dataclass
+class DatasetArtifactUpdate:
+    type: str
+    uri: str
+
+
+@dataclass
+class DatasetData:
+    """
+    Convenience class that returns all the information required by the tests.
+    The ids are already stringified for convenience, since the API layer
+    will work with strings (at least for now)
+    """
+
+    dataset_version_id: str
+    dataset_id: str
+    explorer_url: str
+    collection_version_id: str
+    collection_id: str
+    artifact_ids: List[str]
+
+
+class NewBaseTest(unittest.TestCase):
```

Comment:
Can we rename this to BaseTest? To remove the 'new' since it will stop being 'new' eventually and I'd rather refactor the name out now.